.pre-commit-config.yaml

```diff
@@ -30,6 +30,14 @@ repos:
       - id: mypy
         args: ['--ignore-missing-imports', '--exclude=/setup\.py$']
         additional_dependencies: [types-all]
+
+  - repo: https://github.com/roy-ht/pre-commit-jupyter
+    rev: v1.2.1
+    hooks:
+      - id: jupyter-notebook-cleanup
+        args:
+          # - --remove-kernel-metadata
```

Comment:
can you try to uncomment this line also?

Reply:
Done!